### PR TITLE
Feature/tvist1 510 role permissions

### PR DESCRIPTION
**Pull request by jekuaitk**
* Edited permissions such that only users granted  `ROLE_ADMIN` may add/edit/remove board and municipality entities.
* General cleanup of `/admin`
* Added translations

https://jira.itkdev.dk/browse/TVIST1-510

### CODE REVIEW

**Review by rimi-itk — COMMENTED** [review]
Very good work. A couple of questions asked.

File: translations/case+intl-icu.da.xlf [hunk 1]
```diff
@@ -139,7 +139,7 @@
       </trans-unit>
       <trans-unit id="Auee2PY" resname="Complainant category">
         <source>Complainant category</source>
-        <target>Klagekategori</target>
+        <target>Klagetype</target>
```
[the remark this answers is not in the document]
Comment (jekuaitk) [reply translations/case+intl-icu.da.xlf:142]: `Complainant category` is not something we have and after a search i don't believe it's a translation that is ever used.

My guess is that it was added as a translation as a typo of `Complaint category` that was then later fixed without this translation being removed.

File: translations/admin.da.xlf [hunk 2]
```diff
@@ -11,11 +11,11 @@
       </trans-unit>
       <trans-unit id="SaPHHYy" resname="Complaint category">
         <source>Complaint category</source>
-        <target>Sagstype</target>
+        <target>Klagetyper</target>
```
[the remark this answers is not in the document]
Comment (jekuaitk) [reply translations/admin.da.xlf:14]: Yes. Fixed.

File: translations/case+intl-icu.da.xlf (hunk 1 above)
[the remark this answers is not in the document]
Comment (rimi-itk) [reply translations/case+intl-icu.da.xlf]: https://symfony.com/doc/current/translation/debug.html

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**